### PR TITLE
Adding new DNS for MCM and AuthProxy

### PR DESCRIPTION
**Pull request by aljrubior**

### CODE REVIEW

**Review by mcocaro — CHANGES_REQUESTED** [review]
can you please include a copy text that states that the new DNS endpoints do not have a static IP pls? Thx

File: modules/ROOT/pages/installing-and-configuring-runtime-manager-agent.adoc [hunk 1]
```diff
@@ -855,6 +859,18 @@ There are two static IPs that must be whitelisted to access the mule-manager hos
 |*EU*|*mule-manager.eu1.anypoint.mulesoft.com* |18.194.245.32
 |===
 
+[NOTE]
```
Comment (valkyrie69) [inline modules/ROOT/pages/installing-and-configuring-runtime-manager-agent.adoc:862]: Please change to:

The following DNS endpoints do not have static IP addresses:

**Review by valkyrie69 — CHANGES_REQUESTED** [review]
One minor edit requested.

**Review by mcocaro — APPROVED** [review]
LGTM

**Review by valkyrie69 — APPROVED** [review]

**Pull request merged**